ios/PolyPodApp/PolyPodUITests/UpdateNotificationTest.swift

```diff
@@ -60,6 +60,8 @@ class UpdateNotificationTest: XCTestCase {
             ]
         }
         app.launch()
+        let background = app.wait(for: .runningForeground, timeout: 30)
+        XCTAssertTrue( background )
```

Comment:
The spacing inside parentheses should not be needed, unless there is a preference for it,